mediaqueries-5/Overview.bs

```diff
@@ -1305,94 +1305,73 @@ Display Modes: the ''display-mode'' media feature </h3>
 	Type: discrete
 	</pre>
 
-	The 'display-mode' media feature represents the [=display mode=] of the web application.
+	The 'display-mode' media feature represents the [=display mode=] of the current [=browsing context=].
 	Child browsing contexts reflect the [=display mode=] of their [=top-level browsing context=].
 
-	A <dfn export>display mode</dfn> represents
-	how the web application is being presented within the context of an OS (e.g., in fullscreen, etc.).
-	Display modes correspond to user interface (UI) metaphors
-	and functionality in use on a given platform.
-	The UI conventions of the display modes are purely advisory
-	and implementers are free to interpret them how they best see fit.
+	The values of this feature correspond to the [=display mode|display modes=] defined in
+	[[APPMANIFEST]]:
 
-	This specification defines the following [=display modes=]:
-	<dl dfn-for="display mode" export>
+	<dl dfn-type=value dfn-for="@media/display-mode">
 		<dt><dfn>fullscreen</dfn>
 		<dd>
-			The web application is displayed with browser UI elements hidden
-			and takes up the entirety of the available display area.
+			The [=display mode/fullscreen=] display mode.
 
 		<dt><dfn>standalone</dfn>
 		<dd>
-			The web application is displayed to look and feel like a standalone native application.
-			This can include the application having a different window,
-			its own icon in the application launcher, etc.
-			In this mode,
-			the user agent excludes standard browser UI elements
-			such as an URL bar,
-			but standard system UI elements,
-			such as window decorations, a system status bar, and/or a system back button,
-			remain available.
+			The [=display mode/standalone=] display mode.
 
 		<dt><dfn>minimal-ui</dfn>
 		<dd>
-			This mode is similar to [=display mode/standalone=],
-			but provides the end-user with some means to access
-			a minimal set of UI elements for controlling navigation
-			(i.e., back, forward, reload, and perhaps some way of viewing the document's address).
-			A user agent may include other platform specific UI elements,
-			such as "share" and "print" buttons
-			or whatever is customary on the platform and user agent.
+			The [=display mode/minimal-ui=] display mode.
 
 		<dt><dfn>browser</dfn>
 		<dd>
-			The web application is displayed using the platform-specific convention
-			for opening hyperlinks in the user agent
-			(e.g., in a browser tab or a new window).
+			The [=display mode/browser=] display mode.
 	</dl>
 
+	This media feature reflects the actual display mode that is being used on the current browsing
+	context, not necessarily the one that was requested in the web app manifest (if any).
+
+	On normal web pages, 'display-mode' will have a value of ''browser''. Only pages appearing in the
```

Comment:
I'm not sure we should say anything about "normal web pages" (unless that's actually a thing?).

Reply:
Yeah I wasn't really sure how to express this, but I am trying to simply say (since we're in the context of CSS and not the manifest) that "normally" (on the "drive-by web") this will be "browser".

I've reworded it and put it in a note. The non-normative context I think allows me to talk about "web browsers" as a special class of user agent, which I wouldn't normally talk about in normative text.